### ISSUE TITLE
[Feature Request]: Add Keyboard Configuration Settings

### ISSUE THREAD

**Issue by WeiJian123-tech** [FreeTubeApp/FreeTube#5285]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

FreeTube currently has keyboard shortcuts however does not have a clear way to change each individual keyboard shortcut to another button or combination of buttons. When a user is transitioning from YouTube to FreeTube, they may be used to the keyboard shortcuts of YouTube. 

For example the fast forward buttons for YouTube are:

- L key
- Right Arrow Key
- Ctrl + -> (Windows)
- Spacebar + Shift

Source: [Brave Search Results & AI](https://search.brave.com/search?q=how+to+fast+forward+youtube+shortcut&summary=1&summary_og=e641846be2aa12e8590f91)

This may cause confusion and frustration when the transitioning user is unfamiliar with FreeTube's keyboard shortcuts or does not know about FreeTube's documentation/wiki. This may also be an accessibility issue to users who need to change the keyboard shortcut layout for medical (like hand pain), preferential (like preferring YouTube's layout more), or any other needs.

Adding keyboard configuration settings will improve user experience for all users, new and returning users.

### Proposed Solution

This Feature Request is similar to [#992 ](https://github.com/FreeTubeApp/FreeTube/issues/992). 

The difference is that while [#992 ](https://github.com/FreeTubeApp/FreeTube/issues/992) seeks a toggle option for keyboard shortcuts, this Feature Request asks for a new setting like Theme Settings, Player Settings, or Proxy Settings. 

Within this Keyboard Configuration Settings there could be a few ways to implement this feature. There would be either a drop-down menu (similar to [HTML options tags](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option)) that the user can set their key for the keyboard shortcut. 

### Alternatives Considered

Another way is the user clicks a button which displays a pop-up window where the user would input their key/combination of keys for the keyboard shortcut. Afterwards the pop-up window would close and the new keyboard shortcut given by the user would be saved.

### Issue Labels

ease of use improvement, new feature, new optional setting

**Comment by absidue** [comment]
Duplicate of #251
